ovh/provider.go

```diff
@@ -112,6 +112,8 @@ func Provider() *schema.Provider {
 			"ovh_iam_policy":                                          dataSourceIamPolicy(),
 			"ovh_iam_reference_actions":                               dataSourceIamReferenceActions(),
 			"ovh_iam_reference_resource_type":                         dataSourceIamReferenceResourceType(),
+			"ovh_iam_resourcegroup":                                   dataSourceIamResourceGroup(),
```

Comment:
I am unsire if this should be "ovh_iam_resourcegroup" or "ovh_iam_resource_group".

The API is /v2/iam/resourceGroup So I think both are valid, but "ovh_iam_resource_group" looks better and more readable to me.